public/json/listings/units.json

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
@kaptron did you create this manually? i'm wondering if we can use `save_jasmine_fixtures` in the future so that the fixtures are always automatically regenerated and saved and we don't have to manually make sure to update this in case there are changes to the fields etc. that would be dope.